doc/user/content/release-notes.md

```diff
@@ -50,21 +50,41 @@ Wrap your release notes at the 80 character mark.
 
 {{% version-header v0.5.2 %}}
 
-- Support the [`pg_typeof` function](/sql/functions#postgresql-compatibility-func).
+- Provide the [`list`](/sql/types/list/) type, which is an ordered sequences of
+  homogenously typed elements; they're nestable, too! The type was previously
+  available in v0.5.1, but this release lets you create [`list`s from
+  `text`](/sql/types/list/#text-to-list-casts), making their creation more
+  accessible.
+
+- Support the [`pg_typeof`
+  function](/sql/functions#postgresql-compatibility-func).
+
 - [`COPY TO`](/sql/copy-to) now supports `FORMAT binary`.
+
 - Support the [`DISCARD`](/sql/discard) SQL statement.
-- [`TAIL`](/sql/tail) is now guaranteed to produce output ordered by timestamp.
-- [`TAIL`](/sql/tail) now supports timestamp progress.
-- Syntax for [`TAIL`](/sql/tail) has changed. `WITH SNAPSHOT` is now
-  `WITH (SNAPSHOT)`. `WITHOUT SNAPSHOT` is now `WITH (SNAPSHOT = false)`.
-- Add the `PROGRESSED` option to `TAIL`(/sql/tail).
+
+- [`TAIL`](/sql/tail) now:
+
+  - Produces output ordered by timestamp.
+
+  - Supports timestamp progress with the `PROGRESSED` option.
+
+  - Has slightly different syntax. `WITH SNAPSHOT` is now `WITH (SNAPSHOT)`.
+    `WITHOUT SNAPSHOT` is now `WITH (SNAPSHOT = false)`.
+
 - Report an error without crashing when a query contains unexpected UTF-8
-  characters, e.g., `SELECT ’1’` {{% gh 4755 %}}.
+  characters, e.g., `SELECT ’1’`. {{% gh 4755 %}}
+
 - Log messages are no longer emitted to stderr if an explicit `--log-file` is
   supplied at startup. {{% gh 4777 %}}
-- The systemd package configured by our apt package now writes all logs to
-  the systemd journal, instead of a file in the mzdata directory. {{% gh 4781 %}}
 
+- The systemd package configured by our apt package now writes all logs to the
```

Comment:
tiny nit: "systemd package" isn't a thing. "systemd service" is probably the right thing to say